workflow-templates/upstream-sync.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Upstream Sync
+'on':
+  schedule:
+    - cron: "15 8 * * 1"
```

Comment:
Might as well have a workflow_dispatch here.

Reply:
Good catch. It has now been added.